playbooks/intel_ocp_provisioning.yaml

```diff
@@ -0,0 +1,90 @@
+- hosts: localhost
+  gather_facts: no
+  vars:
+    kubeconfig_path: "~/.kube/config"
+  environment:
+    KUBECONFIG: "{{ kubeconfig_path }}"
+  vars_prompt:
+    - name: "install_operators"
+      prompt: "Do you want to install operators? 'Yes' to install NFD Operator and Intel Device Plugins Operator, or 'No' to skip"
+      private: no
+    - name: "validation_feature"
+      prompt: "Which Intel feature do you want to validate? Enter 1 for Intel SGX, 2 for Intel QAT, 3 for Intel DSA, 4 for Intel GPU"
+      private: no
+  
+  tasks:
+  - name: Validate Inputs
+    block: 
+    - name: Invalid Install Operators Input
+      fail:
+        msg: "Invalid input for Install Operators. Please enter a valid option for Install Operators (Yes/No)."
+      when: install_operators not in ["Yes", "No"]
+    - name: Invalid Validation Feature Input
+      fail:
+        msg: "Invalid input for validation feature. Please enter a valid option (1-4)."
+      when: validation_feature not in ["1", "2", "3", "4"]
```

Comment:
Can user input something like 1,2 to test 1 and 2?

Reply:
For now I have only added 1 or 2 not "and". I will check how we can add 'and'